hoomd/hpmc/ComputeSDF.h

```diff
@@ -264,23 +267,40 @@ template<class Shape> void ComputeSDF<Shape>::zeroHistogram()
     // Zero the histogram
     for (size_t i = 0; i < m_hist.size(); i++)
         {
-        m_hist[i] = 0;
+        m_hist[i] = 0.0;
         }
     }
 
 /*! \param timestep current timestep
 
     countHistogram() loops through all particle pairs *i,j* where *i* is on the local rank, computes
-    the bin in which that pair should be and adds 1 to the bin. countHistogram() can be called
-    multiple times to increment the counters for averaging, and it operates without any
-    communication.
+    the bin corresponding to the scale factor of the vector r_ij that induces the first overlap
+    between particles *i* and *j*, and then adds s_ij to the bin, where s_ij = 1 - e^{\beta\Delta
+   U_{ij}}. Note that this definition allows for the consideration of soft overlaps where the pair
+   potential changes value, and reduces 1 for hard particle overlaps.  countHistogram() operates
+   without any communication.
```

Comment:
```suggestion
    U_{ij}}. Note that this definition allows for the consideration of soft overlaps where the pair
    potential changes value, and reduces to 1 for hard particle overlaps.  countHistogram() operates
    without any communication.
```
Indent with 4 spaces.